src/BulkAppointments/AddPrisoners/AddPrisonersContainer.js

```diff
@@ -97,7 +98,7 @@ class AddPrisonerContainer extends Component {
     return (
       <Page title="Add prisoners" applicationTitle="Whereabouts" alwaysRender>
         {complete && (
-          <div>
+          <ScrollToTop always>
             <H3> {appointmentsCreated} appointments created </H3>
```

Comment:
@govuk-react/heading?

Reply:
At the moment it uses @govuk-react/heading. I just went with what Page.js uses.  I did try and upgrade to use heading as header is decrepcated now but was met with a few issues not resolving 'styled-components'. We need to upgrade to the latest gov components but maybe not for this ticket?

Reply:
Will replace heading with header as it's deprecated.